config/certmanager/certificate.yaml

```diff
@@ -21,4 +21,4 @@ spec:
   issuerRef:
     kind: Issuer
     name: selfsigned-issuer
-  secretName: webhook-server-cert # this secret will not be prefixed, since it's not managed by kustomize
+  secretName: aws-load-balancer-webhook-server-cert # this secret will not be prefixed, since it's not managed by kustomize
```

Comment:
Lets name it aws-load-balancer-webhook-secret since the object is of type secret and will contain info more than certificate.

Reply:
done :D